q2_feature_classifier/classifier.py

```diff
@@ -133,20 +156,24 @@ def classify(reads: DNAIterator, classifier: Pipeline,
 
 
 _classify_parameters = {'chunk_size': Int, 'n_jobs': Int, 'pre_dispatch': Str,
-                        'confidence': Float}
+                        'confidence': Float, 'force_read_direction':
+                        Str % Choices(['forward', 'reverse'])}
 
 
 plugin.methods.register_function(
-    function=classify,
+    function=classify_sklearn,
     inputs={'reads': FeatureData[Sequence],
             'classifier': TaxonomicClassifier},
     parameters=_classify_parameters,
     outputs=[('classification', FeatureData[Taxonomy])],
-    name='Classify reads by taxon.',
+    name='Pre-fitted sklearn-based taxonomy classifier',
     description='Classify reads by taxon using a fitted classifier.',
     parameter_descriptions={'confidence': 'Confidence threshold for limiting '
                             'taxonomic depth. Negative value disables. '
-                            'Currently experimental. USE WITH CAUTION.'}
+                            'Currently experimental. USE WITH CAUTION.',
+                            'force_read_direction': 'Force reads to be treated'
```

Comment:
Can you add a brief description of how the autodetection works? Also, this small addition may be useful (or some variation thereof):

*Force reads to be treated as forward or reverse complement with respect to the reference sequences the classifier was fitted to. [...]*

I'm wondering if a parameter name like `read_orientation` would be a little clearer? It could have a default of None for autodetection. "same" would imply the reads are in the same orientation as the reference sequences, and "reverse-complement" would imply the reads need reverse-complementing to be in the same orientation. I'll defer to you and @gregcaporaso for what you think will be clearest to users.